core/src/main/resources/defaults/clustertemplates/7.2.7/gcp/flow-management.json

```diff
@@ -8,6 +8,9 @@
     "cluster": {
       "blueprintName": "7.2.7 - Flow Management Heavy Duty with Apache NiFi, Apache NiFi Registry"
     },
+    "externalDatabase": {
```

Comment:
is external database supported for GCP? will it be for 7.2.7?

Reply:
> is external database supported for GCP? will it be for 7.2.7?

@pvillard31, i asked Richard about that and he said that Cloudbreak will support external db for GCP in 2.35 Cloudbreak version. According to "Cloudbreak Release Schedule" (https://cloudera.atlassian.net/wiki/spaces/ENG/pages/610469693/Cloudbreak+Release+Schedule), this functionality should be available on prod in the middle of December, on mow-dev it should be available on the beginning of December. So looks like its actual for 7.2.7.

Reply:
LGTM then